### PR TITLE
Add BetterQuest addon kudos @gibbed

### PR DESCRIPTION
**Pull request by zckrs**
Source: https://www.reddit.com/r/treeofsavior/comments/4fa88z/betterquest_an_addonmod_for_the_quest_ui/

### CODE REVIEW

File: addons/betterquest/betterquest.lua [hunk 1]
```diff
@@ -67,3 +67,7 @@ function BETTERQUEST_POSSIBLE_UI_OPEN_CHECK(pc, questIES)
 
 	return 'HIDE';
 end
+
+BETTERQUEST_ON_INIT();
+
+ui.SysMsg("Better Quest loaded!");
```
Comment (zckrs) [inline addons/betterquest/betterquest.lua]: @gibbed like this ?

Comment (gibbed) [inline addons/betterquest/betterquest.lua]: That will work.

**Comment by Excrulon** [comment]
This isn't my addon. Not just going to add it unless @gibbed wants it bundled with this.

In the near future we'll have a well defined standardized system (not using my addonloader) where everyone can easily contribute and download addons. Not sure on the eta of that but maybe in a week or two.

**Comment by zckrs** [comment]
@gibbed are you agree to add your add-on in this bundle?

**Comment by pedrosotero** [comment]
Tested it works

**Comment by Miei** [comment]
Checkbox disappears after a loading screen for me, can be fixed by adding:

```
function QUEST_ON_INIT_HOOKED(addon, frame)
	_G["QUEST_ON_INIT_OLD"](addon, frame);
	BETTERQUEST_ON_INIT();
end

local questOnInitHook = "QUEST_ON_INIT";

if _G["QUEST_ON_INIT_OLD"] == nil then
	_G["QUEST_ON_INIT_OLD"] = _G[questOnInitHook];
	_G[questOnInitHook] = QUEST_ON_INIT_HOOKED;
else
	_G[questOnInitHook] = QUEST_ON_INIT_HOOKED;
end

**Comment by zckrs** [comment]
@Miei just added

**Comment by zckrs** [comment]
Thank for your comments

I squash commit to get only one. But I am not sur @Excrulon will merge this PR :anguished:

**Comment by rncoll7** [comment]
https://www.reddit.com/r/treeofsavior/comments/4fa88z/betterquest_an_addonmod_for_the_quest_ui/d2esqu0

**Comment by Excrulon** [comment]
He's completely ignoring me on this so not going to bother. Will probably be writing my own quest addon soon.

**Comment by zckrs** [comment]
@Excluron stop to comment and delete your comment xD.  Do you want I update the script?

**Comment by zckrs** [comment]
:tada:

**Comment by fiote** [comment]
It's weird to have betterquest on your repository lol. Are you going to maintain it from now on?

**Comment by Excrulon** [comment]
He said I could merge it. When he puts it up on his own github and the
addon manager is finished, we can point it to his.

There are a few things I want to add to it if I ever get time.

**Comment by fiote** [comment]
Sounds fair enough. Thanks for the feedback!

**Pull request merged**